catalog/USGS/USGS_NLCD_RELEASES_2019_REL_RCMAP_V5_TRENDS.jsonnet

```diff
@@ -0,0 +1,327 @@
+local id = 'USGS/2019_REL/RCMAP/V5/TRENDS';
+local subdir = 'USGS';
+
+local ee_const = import 'earthengine_const.libsonnet';
+local ee = import 'earthengine.libsonnet';
+local spdx = import 'spdx.libsonnet';
+local license = spdx.cc0_1_0;
+
+local basename = std.strReplace(id, '/', '_');
+local base_filename = basename + '.json';
+local self_ee_catalog_url = ee_const.ee_catalog_url + basename;
+local self_url = catalog_subdir_url + base_filename;
+
+{
+  stac_version: ee_const.stac_version,
+  type: ee_const.stac_type.collection,
+  stac_extensions: [
+    ee_const.ext_eo,
+    ee_const.ext_sci,
+    ee_const.ext_ver,
+  ],
+  id: id,
+  title: 'RCMAP Rangeland Component Timeseries V5 Trends (1985-2021)',
+  version: 'V5',
```

Comment:
Use the local variable I suggested before. And `V` is not a part of the version number.

`'V5'` -> `version`